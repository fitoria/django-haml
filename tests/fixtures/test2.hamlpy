%ul#athletes
    - for athlete in athlete_list
        %li.athlete{'id': 'athlete_{{ athlete.pk }}'}= athlete.name