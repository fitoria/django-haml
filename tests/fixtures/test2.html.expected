<ul id='athletes'>
    {% for athlete in athlete_list %}
        <li id='athlete_{{ athlete.pk }}' class='athlete'>{{ athlete.name }}</li>
    {% endfor %}
</ul>
